waterlog/src/Components/AlertsMap.js

```diff
@@ -1,11 +1,21 @@
 import React from 'react';
 import Loader from './Loader';
+import MapComponent from './Map';
 
 const AlertsMap = (props) => {
-    return (
-      <div className="alerts-map">
-        <Loader />
-      </div>
-    )
-}
-export default AlertsMap;
\ No newline at end of file
+	const { error, loading, sensorLocations } = props;
```

Comment:
You can place this inside the function parameter list.
```js
const AlertsMap = ({ error, loading, sensorLocations })
```

Reply:
I placed it in the function parameter list.